lab_4_retrieval_w_clustering/main.py

```diff
@@ -36,10 +41,13 @@ class BM25Vectorizer(Vectorizer):
     _corpus: TokenizedCorpus
     _avg_doc_len: float
 
-    def __init__(self) -> None:
+    def __init__(self, corpus: TokenizedCorpus) -> None:
```

Comment:
I think you should not change the interface

Reply:
fixed